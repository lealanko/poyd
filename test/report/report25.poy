(* Build and report trees : total *)
read ("1.fas", "30.ss")
set(seed:50)
build (1)
report (trees: (total))
