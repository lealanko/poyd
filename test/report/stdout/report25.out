(t1 (t2 ((t27 (t26 t25)) ((t29 t28) ((t40 ((t11 (t47 t38)) (t6 (t20 t18))))
 ((t49 (t33 (t4 (t5 (t7 t34))))) (((t32 t3) (t9 (t8 t36))) (((t16 (t15 (t45
 t35))) (t37 ((t48 t13) (t14 t12)))) (t46 (t39 (t10 (t17 (t42 (t31 ((t23 (t24
 t21)) (t22 ((t44 t43) (t41 (t30 t19)))))))))))))))))))[2335.];